frontend/src/components/Forms/EditTailingsReportForm.js

```diff
@@ -59,6 +61,15 @@ export const EditTailingsReportForm = (props) => (
             validate={[]}
           />
         </Form.Item>
+        <Form.Item label="Upload Documents">
+          <Field
+            id="tsf_document_upload"
+            name="tsf_document_upload"
+            expDocumentGuid={props.expDocumentGuid}
+            component={FileUpload}
+            validate={[]}
```

Comment:
I think the validate attribute can be removed if no validations are applied

Reply:
This is a fair point, but the existing items on the page all have empty validators. I could change all of them?

Reply:
Yeah remove them all.

Reply:
Empty validators have been removed.